qa/workunits/rbd/nvmeof_fio_test.sh

```diff
@@ -19,18 +55,24 @@ size=${SIZE:-1G}
 time_based=1
 runtime=$RUNTIME
 rw=${RW:-randrw}
-filename=$(echo "$drives_list" | tr '\n' ':' | sed 's/:$//')
+filename=$(echo "$selected_drives" | tr '\n' ':' | sed 's/:$//')
 verify=md5
 verify_fatal=1
+direct=1
 EOF
 
-fio --showcmd $fio_file
-sudo fio $fio_file &
+echo "[nvmeof] starting fio test..."
 
 if [ -n "$IOSTAT_INTERVAL" ]; then
     iostat_count=$(( RUNTIME / IOSTAT_INTERVAL ))
-    iostat -d $IOSTAT_INTERVAL $iostat_count -h 
+    iostat -d -p $selected_drives $IOSTAT_INTERVAL $iostat_count -h &
 fi
+if [ "$rbd_iostat" = true  ]; then
+    iterations=$(( RUNTIME / 5 ))
+    rbd perf image iostat mypool --iterations $iterations &
```

Comment:
Hard coding the pool name which is otherwise parametrized isn't nice.  If you don't expect this command to be replaced with something more reliable soon, perhaps add `RBD_POOL` environment variable similar to `nvmeof_setup_subsystem.sh`?